activities/communication/run-a-community-call.md

```diff
@@ -0,0 +1,108 @@
+---
+type: Resource
+---
+
+# How to run a community call
+
+The roles on the community call are:
+
+* the chair, who opens the call on behalf of the Foundation
+* experts leading specific substantive sections of the call (otherwise this is the chair)
+* the notetaker
+
+## Before the call
+
+The chair and the notetaker:
+
+* create a notes HackMD.io document for this call
+* agree the Foundation update points (since the last call or last demo - ask the [communications coordinator](../../organization/staff.md#communications) if in doubt)
+* put these in the call notes doc
+* add "updates since the last community call" to the notes - keep this to three items or less (we don't want the community call to be too much of a reporting channel). If possible, focus on community related updates
+
+The notetaker will take notes during the call in the same document.
+
+### Conference call etiquette
+
+Each person should use their own computer so everyone on the call is symbolically equal.
+
+The chair should be on the call 5 min before the call start.
+
+Consider putting a Foundation poster on the wall behind you.
+
+## Templated agenda items
+
+1. Opening the call
+
+* [ ] Introduction round (All)
+* [ ] Short update from the Foundation for Public Code (Jan)
+
+2. Substance of the call
+
+
+## 1. Opening the call
+
+Call-in link: https://meet.google.com/tfd-acmn-tkb
+Agenda link: https://hackmd.io/-OegeqvoThCbAsw3c3gIjw?edit
```

Comment:
Could these be actual links, it helps processing and printing markdown if we have a consistent way of linking to other places.

Reply:
Yes, they could and that is a good suggestion. (These kind of links are by the way the reason I suggested we should get ourselves a URL shortener so that they also would be easy for a human to reproduce.)